lib/draw/BpmnRenderUtil.js

```diff
@@ -14,6 +14,8 @@ import {
 // re-export getDi for compatibility
 export { getDi };
 
+export var black = 'hsl(225, 10%, 15%)';
```

Comment:
Should we base this of a CSS variable so custimizing is possible? Just wondering.

Reply:
I thought we did it in other places, too (for theming). But never mind; cannot find these anymore.

Reply:
I was thinking about that. But on the other hand, the color is already configurable.

Reply:
Ah exactly. We can configure it via renderer config. Thanks!